src/colony/modules/net.js

```diff
@@ -195,9 +195,11 @@ TCPSocket.prototype.connect = function (/*options | [port], [host], [cb]*/) {
         var closeRet = tm.tcp_close(self.socket); // returns -57 if socket is already closed
         if (closeRet < 0 && closeRet != -tm.ENOTCONN){ 
           // couldn't close socket, throw an error
+          // failed to connect, stay silent
+          console.log("Failed to close socket", self.socket);
```

Comment:
But yeah, should we have internal code be emitting console.logs if it's just going to throw the error immediately?